### ISSUE TITLE
Whether can support html or vue script tag

### ISSUE THREAD

**Issue by jing2si** [RobinMalfait/prettier-eslint-code#8]
All in title.

**Comment by ryh** [comment]
Yeah, I got this [Language is not valid](https://github.com/RobinMalfait/prettier-eslint-code/blob/master/src/utils.ts#L73) as well (for .vue file).
And I already put plugin into `.eslintrc.js`
```js
plugins: [
    'html','vue'
  ],
```

**Comment by RobinMalfait** [comment]
So the extension is `.vue` are there other extensions?
I don't know if `.html` is a good plan though... 🤔

**Comment by ryh** [comment]
there are great variety file extension for users , for example I use `.vue`  (Vue) `.blade.php` (Laravel Blade) but some may not.  I suggest add a config for user to define the `VALID_LANG` 

what I tried

- copy the js in .vue into a standalone .js file then use the extension , worked well.
- clone this repo and change `const VALID_LANG: DocumentSelector = ['javascript', 'javascriptreact', 'html', 'vue'];`, then vsce package and install . worked in selecting the js code and prettier. but reported `prettier-eslint [ERROR]: prettier formatting failed due to a prettier error` while using the prettier to the .vue document (without selecting)

**Comment by RobinMalfait** [comment]
The reason it errors, is because there are multiple 'languages' in the file. So prettier takes in the whole file (or selection) but it sees html tags like <script> and it fails. We just have to figure out a way to detect `<script/>` tags inside the page, if it is a `.vue` or `.html` file.
